config.yml

```diff
@@ -0,0 +1,2 @@
+input_file_path: ../input.xlsx
```

Comment:
Don't forget that the name of the input nt user sheet must also be given here :)

Also, I think that in practice, the `input_file_path` and `output_file_path` will be the same. But we can keep 2 separate config settings.